content/setting-up-services/how-to-set-up-services/running-a-background-process.md

```diff
@@ -22,7 +22,7 @@ You can use two techniques to run a background process:
 Our [prebuilt images](../../service-images/using-tugboat-images/) use [runit](http://smarden.org/runit/) to start and
 manage background processes.
 
-To add your own background process that starts when the Service starts, create a directory in
+To add your own background process that starts when the Service starts, create a directory
```

Comment:
Thoughts on this @jsoref ?

```suggestion
To add your own background process that starts when the Service starts, create the directory
```

Reply:
Offhand, I don't think so, but I can't quite explain why. 

I think it's because it isn't a specific directory, it's just _a_ directory. If it were _the_ specific directory `/etc/service`, then I'd use `the`.

Reply:
Makes sense!